src/main/java/io/cryostat/JsonRequestFilter.java

```diff
@@ -39,11 +39,11 @@ public class JsonRequestFilter implements ContainerRequestFilter {
     static final Set<String> disallowedFields = Set.of("id");
     static final Set<String> allowedPathPatterns =
             Set.of(
-                    "/api/v2.2/discovery",
-                    "/api/v2/rules/[\\w]+",
+                    "/api/v4/discovery",
+                    "/api/v4/rules/[\\w]+",
                     "/api/beta/matchExpressions",
-                    "/api/v2.2/graphql",
-                    "/api/v3/graphql");
+                    "/api/v2/graphql",
```

Comment:
`v2/graphql` can be removed now, right?

Reply:
yes

Reply:
v1 path should also be v4? @andrewazores

Reply:
Yes please, everything should become v4 in this release.